packages/react-lowcode/src/codegen/ast/node.ts

```diff
@@ -22,6 +22,15 @@ export const isOpeningOrSelfClosingElementWithName = (node: ts.Node, name: strin
     }
 }
 
+export const isUseQueryHookImport = (node: ts.Node, name: string): boolean | undefined => {

```

Comment:
Maybe better place for this could be ~/codegen/ast/imports alebo hooks

Reply:
Moved functions to `imports` and `hooks` folders